docs/pages/cardinal/ecs.mdx

```diff
@@ -0,0 +1,42 @@
+# Introduction to ECS
+
+Cardinal follows the ECS (Entity-Component-System) architecture which facilitates fast and efficient updates to game
+state, as well as enabling painless extensibility to your games.
```

Comment:
We shouldn't do line breaks on markdown like this IMO. It's better to just let the IDE and GitHub softwraps it